src/main/java/net/sf/mzmine/modules/peaklistmethods/alignment/adap3/help/help.html

```diff
@@ -0,0 +1,78 @@
+<!--
+  ~ Copyright (C) 2019 Du-Lab Team <dulab.binf@gmail.com>
+  ~
+  ~ This program is free software; you can redistribute it and/or
+  ~ modify it under the terms of the GNU General Public License
+  ~ as published by the Free Software Foundation; either version 2
+  ~ of the License, or (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program; if not, write to the Free Software
+  ~ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+  -->
+
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+        "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+    <title>ADAP Alignment</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <link rel="stylesheet" type="text/css" href="/net/sf/mzmine/desktop/impl/helpsystem/HelpStyles.css">
+</head>
+<body>
+
+<h1>ADAP Aligner</h1>
+
+<h2>Description</h2>
+
+<p>
+    This method aligns detected compounds in different samples through matching their mass fragmentation spectra and
```

Comment:
aligns compounds -> aligns peaks

Reply:
I think that using the term "peaks" for this algorithm may be confusing. Unlike Join Aligner and RANSAC Aligner that align peaks (i.e. entities with m/z and retention time values), this algorithm aligns compounds (i.e. entities with fragmentation mass spectrum and retention time). Would you still prefer using term "peaks" in this case?

Reply:
"Features" will work, thanks.

Reply:
I've changed all "compounds" to "features"